sensirion_arch_config.h

```diff
@@ -38,6 +38,16 @@
  */
 #include <stdint.h>
 
+/**
+* If your platform does not provide the library stdlib.h you have to
+* define NULL yourself (see below).
+ */
+#include <stdlib.h>
```

Comment:
If there is no stdlib header file, this include won't work

Reply:
Right, see the comment above.

Reply:
But maybe change it to: 
```
/**
* If your platform does not provide the library stdlib.h you have to remove the include and
* define NULL yourself (see below).
 */
```

Reply:
Done.